### PR TITLE
Move organization login authenticator from org management repo.

### PR DESCRIPTION
**Pull request by wenodpathirana**

### CODE REVIEW

File: .github/workflows/pr-builder.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ env.cache-name }}-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-${{ env.cache-name }}-
+            ${{ runner.os }}-maven-
+            ${{ runner.os }}-
+      - name: Build with Maven
+        run: mvn clean install -U -B
+      - name: Delete SNAPSHOT artifacts
+        run: find ~/.m2/repository/ -name "*-SNAPSHOT" -type d -print -prune -exec rm -r {} +
```
Comment (dewniMW) [inline .github/workflows/pr-builder.yml:42]: new line

File: .gitignore [hunk 2]
```diff
@@ -21,3 +30,8 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+# Ignore everything in this directory
+target/
+
+.DS_Store
```
Comment (dewniMW) [inline .gitignore:37]: new line

File: components/org.wso2.carbon.identity.application.authenticator.organization.login/pom.xml [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+                        <goals>
+                            <goal>report-integration</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+</project>
```
Comment (dewniMW) [inline components/org.wso2.carbon.identity.application.authenticator.organization.login/pom.xml:224]: new line

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,391 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2021, WSO2 LLC. (http://www.wso2.com).
```
Comment (dewniMW) [inline pom.xml:3]:
```suggestion
  ~ Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
```

**Review by dewniMW — APPROVED** [review]

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,392 @@
...
+        <findbugs.annotations.version>3.0.1</findbugs.annotations.version>
+        <carbon.p2.plugin.version>5.1.2</carbon.p2.plugin.version>
+        <maven.buildnumber.plugin.version>1.4</maven.buildnumber.plugin.version>
+
+        <compiler-source.version>1.8</compiler-source.version>
+        <compiler-target.version>1.8</compiler-target.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+</project>
+`
```
Comment (dewniMW) [inline pom.xml:392]:
```suggestion

```

**Review by dewniMW — APPROVED** [review]

File: pom.xml (hunk 5 above)
Comment (wenodpathirana) [reply pom.xml]: fixed

**Pull request merged**